client/src/Components/local/atoms/core-mapping/helper/map-controls.tsx

```diff
@@ -40,8 +40,9 @@ const MapControls: React.FC<GeomanControlProps> = ({ onCreate, onShowLabels }) =
 
         layersVisible = !layersVisible 
       },
-
-      toggle: true
+      
+      // Set toggle to false to indicate that this custom control does not have a toggle functionality
```

Comment:
but @lilitkarapetyan  this *should* have toggle behaviour, since we're switching on and off the symbol labels.

Reply:
switching to true results in the loss of the layer name when transiting to deferent symbols (https://gyazo.com/f4bcf1eede84994a3b88ebef2f572c58) that's right ?

Reply:
I can investigate it

Reply:
I'm a couple of commits behind you Lilit, but it seems ok to me:
https://i.gyazo.com/8426bcfa1615adb5d19a2d9de51dcc82.mp4